### PR TITLE
Add tox-based Integration Testing Support

### PR DESCRIPTION
**Pull request by peterallenwebb**
Resolves: #109 

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (emmyoop) [inline .github/workflows/ci.yml:33]:
```suggestion
    #     DBT_ENV_SECRET_REDSHIFT_PASS: ${{ secrets.DBT_ENV_SECRET_REDSHIFT_PASS }}

```

File: integration_tests/.env/postgres.env
Comment (emmyoop) [inline integration_tests/.env/postgres.env:6]:
```suggestion
POSTGRES_SCHEMA=audit_helper_integration_tests_postgres

```

File: supported_adapters.env
Comment (emmyoop) [inline supported_adapters.env:1]:
```suggestion
SUPPORTED_ADAPTERS=postgres

```

File: tox.ini
Comment (emmyoop) [inline tox.ini:24]:
```suggestion
    dbt debug --target postgres

```

**Review by emmyoop — COMMENTED** [review]
Looks to me that this covers everything to get postgres tests working!  Just some nits on adding empty lines at the end of the files.

I'd like to get approval from a repo owner before merging.

**Review by joellabes — APPROVED** [review]
lgtm!

**Pull request merged**
